tests/utils/test_cache.py

```
@@ -129,28 +141,143 @@ def test_redis_creation_failure(self, mock_redis_class, mock_get_creds, mock_red
+        mock_client = Mock()
+        mock_redis_class = Mock(return_value=mock_client)
+        monkeypatch.setattr("util.cache.redis.Redis", mock_redis_class)
+
+        mock_client.ping.return_value = True
+
+        client = RedisCache()
+
+        mock_redis_class.assert_called_once_with(
+            host="localhost",
+            port=6380,
+            password="dev-pass",
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to argument: "password"

(S106)